tools/database/wowhead_tooltips.go

```diff
@@ -160,6 +160,7 @@ var spiritRegex = regexp.MustCompile(`<!--stat6-->\+([0-9]+) Spirit`)
 var staminaRegex = regexp.MustCompile(`<!--stat7-->\+([0-9]+) Stamina`)
 var spellPowerRegex = regexp.MustCompile(`Increases spell power by ([0-9]+)\.`)
 var spellPowerRegex2 = regexp.MustCompile(`Increases spell power by <!--rtg45-->([0-9]+)\.`)
+var masteryRegex = regexp.MustCompile(`<!--rtg49-->([0-9]+)\.`)
```

Comment:
Is this regex correct? Shouldnt it look for mastery keyword or something like that or is mastery just a +value on tooltips with no extra description?

Reply:
The item tooltip shows mastery as `<span class=\"q2\">+<!--rtg49-->216 Mastery</span>` for example.

Reply:
That would make the Regex `<!--rtg49-->([0-9]+)\ Mastery.`

Reply:
Fixed in [2445152](https://github.com/wowsims/cata/pull/1/commits/244515279e40ca942b434e2da279ac72ab1743d5)